assets/css/v2/pre_fare/simulation.scss

```diff
@@ -24,13 +24,40 @@
   .simulation {
     transform: scale(17.92%);
     transform-origin: top left;
+
+    --panel-width: 1080px;
+
+    &--duo {
+      width: calc(var(--panel-width) * 2);
+    }
+
+    &--left,
+    &--right,
+    &--solo {
+      width: var(--panel-width);
+    }
+
+    &--right > *,
+    &--solo > * {
+      margin-left: calc(var(--panel-width) * -1);
+    }
   }
 
   .simulation__full-page {
     z-index: 999;
-    max-width: 388px;
     max-height: 407px;
-    margin: 16px;
+
+    --width: 200px;
+
+    &--duo {
+      width: calc(var(--width) * 2);
```

Comment:
What's the reason we need to set a width on this, separately from `.simulation`? It seems redundant, and fragile (assuming the width set here needs to line up with the width above multiplied by the transform scale, though this is not _quite_ true of the new value you've set it to).

Reply:
Ah this was leftover code from me trying to fix the issue with the left side and right side pages not showing up as individual items in the simulation.

I was able to get the right side to show up (by changing some filtering and length logic in `simulation_screen_container.tsx`) and this styling was just to handle some of the extra white space due to the solo layout. None of that really for anything more than just me playing around to try and understand the code base.

I eventually took out the code because I wasn't quite sure what the logic of the flex space and the rightScreenPage logic, but I forgot to take this width part out.